app/utils.py

```diff
@@ -4,16 +4,35 @@
 from enum import Enum
 from types import GeneratorType
 from boardgamegeek import BGGClient, CacheBackendSqlite
-from schema import Game
+from app.schema import Game
 import datetime
 import json
+from itertools import islice
+
+
+def take(n, iterable):
+    "Return first n items of the iterable as a list"
+    return list(islice(iterable, n))
```

Comment:
I think it's probably appropriate to just say `iterable[:n]` wherever you might use this function. Is there a particular use-case you have in mind?

Reply:
I initially was using it when setting up my test data. I was importing a json file containing 700+ items, but only wanted to put 25 in the table. I found that I needed `islice` or to build a list to subscript the result of `json.load`. Maybe would actually have wanted to only read 25 items from the file instead of loading the whole thing and then truncating? Anyway I ended up creating a new file with only 25 items, so not actually using this anymore